src/modules/loki/geometry/Calibration.h

```diff
@@ -21,31 +23,56 @@
 namespace Loki {
 class Calibration {
 public:
+
   Calibration();
 
-  /// \brief Create pixelmappings from calibration file
-  /// reads a vector of uint32_t and updated the number of pixels
+  /// \brief populate the strawcalibration vector with the provided
+  /// polynomial coefficients
   Calibration(std::string CalibrationFile);
 
-  /// /brief Create a null 'Calibration' no pixel mapping is done
+  /// \brief Create a null 'Calibration' no pixel mapping is done
   void nullCalibration(uint32_t Straws, uint16_t Resolution);
 
   /// \brief return the maximum pixel id
   uint32_t getMaxPixel() { return MaxPixelId; }
 
 
-  uint16_t strawCorrection(uint16_t Straw, uint16_t Pos) {
-    return StrawMapping[Straw][Pos];
+  /// \brief apply the position correction
+  uint32_t strawCorrection(uint32_t StrawId, double Pos) {
```

Comment:
Is there a special reason for this being in the `.h` file instead of in the `.cpp` file?